src/pages/chainlink-nodes/v1/using-forwarder.md

```diff
@@ -0,0 +1,353 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: nodeOperator
+date: Last Modified
+title: "Forwarder tutorial"
+metadata:
+  title: "Chainlink Node Operators: Forwarder tutorial"
+  description: "Use a Forwarder contract for more security and flexibility."
+setup: |
+  import CodeSample from "@components/CodeSample/CodeSample.astro"
+  import ClickToZoom from "@components/ClickToZoom.astro"
+---
+
+:::note[Prerequisites]
+This guide assumes you have a running Chainlink node. See the [Running a Chainlink Node locally](/chainlink-nodes/v1/running-a-chainlink-node) guide to learn how to run a Chainlink node. **Note**: For this example to work, you must use Chainlink node version _1.12.0_ or above.
+
+Also, you must be familiar with these concepts:
+
+- [Forwarder](/chainlink-nodes/contracts/forwarder) contracts.
+- [Operator](/chainlink-nodes/contracts/operator) contracts.
+- [OperatorFactory](/chainlink-nodes/contracts/operatorfactory) contracts.
+  :::
+
+<ClickToZoom src='/images/chainlink-nodes/node-operators/forwarder/forwarder-directrequest-example.png' />
+
+In this tutorial, you will configure your Chainlink node with a simple transaction-sending strategy on the Sepolia testnet:
+
+- Your node has two externally owned accounts (EOA).
+- Your node has two [direct request](/chainlink-nodes/oracle-jobs/job-types/direct_request) jobs. One job returns _uint256_, and the other returns _string_.
+- Each job uses a different EOA.
+- You use a [forwarder](/chainlink-nodes/contracts/forwarder) contract to fulfill requests with two EOAs that look like a single address.
+
+:::note
+Here you are using a forwarder contract and two EOAs for two [direct request](/chainlink-nodes/oracle-jobs/job-types/direct_request) jobs. You can use the same strategy for different job types, VRF and OCR. Supporting different job types on the same Chainlink node reduces your infrastructure and maintenance costs.
+:::
+
+## Check your Chainlink node is running
+
+On your terminal, check that your Chainlink node is running:
+
+```shell
+ docker ps -a -f name=chainlink
+
+ CONTAINER ID   IMAGE                            COMMAND               CREATED         STATUS                   PORTS                    NAMES
+ feff39f340d6   smartcontract/chainlink:1.12.0   "chainlink local n"   4 minutes ago   Up 4 minutes (healthy)   0.0.0.0:6688->6688/tcp   chainlink
+```
+
+Your Chainlink Operator Interface is accessible on [http://localhost:6688](http://localhost:6688).
```

Comment:
Should we include a reminder for how to create the SSH tunnel? I copy and paste the command from the docs frequently, so I find it very useful.

Reply:
good idea. check last commit